LBH.AdultSocialCare.Api/V1/Boundary/DayCarePackageOpportunityBoundary/Request/DayCarePackageOpportunityForUpdateRequest.cs

```diff
@@ -0,0 +1,9 @@
+namespace LBH.AdultSocialCare.Api.V1.Boundary.DayCarePackageOpportunityBoundary.Request
+{
+    public class DayCarePackageOpportunityForUpdateRequest
```

Comment:
I wonder if the creation and update objects can be merged to DayCarePackageOpportunityRequest?

Reply:
> I wonder if the creation and update objects can be merged to DayCarePackageOpportunityRequest?

It is possible we can have more fields added to or removed from either the create or update request. Left with a single responsibility that change would be very easy. And also required fields may be different in create and update